### ISSUE TITLE
Facing some issues during integration

### ISSUE THREAD

**Issue by mbaqir** [Kandy-IO/kandy-laravel#3]
Hi Guys,

I am facing some issues while integrating this kandy plugin with Laravel 5.1. I have made a page where I am rendering video call widget of Kandy. It shows two screens named as "Them" and "Me". Also, there is a dropdown on top listing other users to whom  you can make a call. Now I have two questions:

- For Example, I select a user  "User2" from the dropdown, and press "Call" button, How will user2 receive the call? From where, he can be online? 
- Secondly, the window labeled as "Me" is not loading my video. How can this issue be resolved?

I am using this kandy plugin to implement live peer-to-peer video call. As far as I have studied, it will work for me if above two issues are resolved.

Waiting for a quick response.

Thanks,
Muhammad Baqir

**Comment by kodeplusdev** [comment]
@mbaqir I assume you are using Chrome. Can you press F12 to show developer tool and show me any errors in the log and you saw. Thanks!

**Comment by mbaqir** [comment]
@kodeplusdev  No errors are present in the console tab. Can you please answer my above two questions? Waiting for a quick response

**Comment by kodeplusdev** [comment]
@mbaqir 

Answering your 2 questions:

 - For user2. you can open a new Incognito window and login as user2.

- It works fine for me so not sure what the issue you got. You could refer to the complete demo app below. We wrote it as an example of how to use Kandy laravel package.

Demo app:

https://bitbucket.org/kodeplusdev/kandy-laravel-demo/branches/

## Some notes:
- If you want to use Video calls, you would need to use HTTPS
- There are 2 kinds of users: laravel users and kandy users. When you sync users, it will automatically get users from Kandy portal and insert to your local database. You would need to do an extra step to assign laravel users to kandy users. Only laravel user who is assigned to kandy user would be able to use the feature like call, chat,...

If you need further help, you could skype me at `dkhoa81`. Thanks!

**Comment by mbaqir** [comment]
@kodeplusdev  Thanks for a very detailed response. I will ping you over Skype if needed

**Comment by mbaqir** [comment]
@kodeplusdev  Thanks for your guidance. I have deployed your demo app on my linux server. But I am getting errors while attempting video/voice calls. I get an error "Call Failed" from kandy.js file. 

Url: http://45.55.75.47/kandy-laravel-demo/public/index.php

Users:
user1@gmail.com / 111111
user2@gmail.com / 111111

Is it due to Https ? because https is not configured on my server

Looking forward for a quick response.

Thanks

**Comment by kodeplusdev** [comment]
@mbaqir Yes, as you can see in the message, it requires HTTPS for video call

![image](https://cloud.githubusercontent.com/assets/7855608/13734138/6c8ebc7c-e9cd-11e5-9967-2c0977effd26.png)

**Comment by mausumi744** [comment]
hi.
would u like to share your demo code??
that would really save my life.

@ghost
